src/plugins/addLoadingFlag.js

```diff
@@ -2,12 +2,13 @@ import { wrap, Chart } from 'highcharts/highstock';
 
 export default () => {
     wrap(Chart.prototype, 'showLoading', function showLoading(proceed, ...args) {
-        this.isLoading = true;
+        if (Object.keys(this).length === 0) return;
```

Comment:
Additional information

when `showLoading` is called, this plugin code will be trigger twice, the first time is correct, the second is somewhat wrong as `this` became empty object, which throw inside highchart code